fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/DownloadRobot.kt

```diff
@@ -163,6 +164,21 @@ class DownloadRobot {
     fun clickMultiSelectRemoveButton() =
         itemWithResIdContainingText("$packageName:id/title", "Remove").click()
 
+    fun openPageAndDownloadFile(url: Uri, downloadFile: String) {
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(url) {
+            Log.i("MozTestLog", "Entered and submited $url to search bar")
```

Comment:
I think the log would be better placed inside the enterUrl method, so it shows up anytime you load a page in any test, without re-adding the log.

Same for the ones below: Log.i("MozTestLog", "Waited for $url to load") - should be inside waitForPageToLoad()

Reply:
Sounds good, in this case, I'll add the logs in the functions.

@jjSDET , I've had a chat with @sv-ohorvath and weren't sure if we would want to use Log.i or Log.d ? what would you suggest ? ☺️

Reply:
Made the changes, added the log inside the functions called in `openPageAndDownloadFile`